src/Transformer/FractalTransformer.php

```diff
@@ -76,7 +76,9 @@ public function transform($response, $transformer, Binding $binding, Request $re
         }
 
         if ($response instanceof EloquentCollection) {
-            $response->load($this->fractal->getRequestedIncludes());
+            $requestedIncludes = $this->fractal->getRequestedIncludes();
+            $eagerLoads = $this->getEagerLoads($transformer, $requestedIncludes);
+            $response->load($eagerLoads);
```

Comment:
Didn't even see this comment. My bad.

But that's the whole point of this. Is to eagerly load any requested includes. Otherwise they are lazily loaded. This was initially brought up in #118.